Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Tcc_Feature.dlt

```diff
@@ -20,6 +20,16 @@ SILICON_CFG_DATA.EnableTimedGpio1 | 1
 
 # Enable RTCM from the boot option
 # Yocto Mender (BIT4), PreOs (BIT2) and RTCM (BIT3)
-BOOT_OPTION_CFG_DATA_0.BootFlags_0 | 28
+BOOT_OPTION_CFG_DATA_0.BootFlags_0 | 28          #USB
+#BOOT_OPTION_CFG_DATA_3.BootFlags_3 | 28         #SATA
+#BOOT_OPTION_CFG_DATA_6.BootFlags_6 | 28         #MMC
+#BOOT_OPTION_CFG_DATA_9.BootFlags_9 | 28         #NVME
+#BOOT_OPTION_CFG_DATA_12.BootFlags_12 | 28       #UFS1
+#BOOT_OPTION_CFG_DATA_15.BootFlags_15 | 28       #UFS2
 # Set boot option 2 (RTCM) as Extra image type
-BOOT_OPTION_CFG_DATA_2.ImageType_2 | 0x1E
+BOOT_OPTION_CFG_DATA_2.ImageType_2 | 0x1E        #USB
+#BOOT_OPTION_CFG_DATA_5.ImageType_5 | 0x1E       #SATA
+#BOOT_OPTION_CFG_DATA_8.ImageType_8 | 0x1E       #MMC
+#BOOT_OPTION_CFG_DATA_11.ImageType_11 | 0x1E     #NVME
+#BOOT_OPTION_CFG_DATA_14.ImageType_14 | 0x1E     #UFS1
+#BOOT_OPTION_CFG_DATA_17.ImageType_17 | 0x1E     #UFS2

```

Comment:
Instead of putting all same options in boot option, why don't you change BootDeviceType in 1st boot option?
```
ex) To use NVMe
BOOT_OPTION_CFG_DATA_0.BootFlags_0 | 28
BOOT_OPTION_CFG_DATA_0.BootDeviceType_0 | 6
BOOT_OPTION_CFG_DATA_2.ImageType_2 | 0x1E
BOOT_OPTION_CFG_DATA_2.BootDeviceType_2 | 6
or
ex) To use SATA
BOOT_OPTION_CFG_DATA_0.BootFlags_0 | 28
BOOT_OPTION_CFG_DATA_0.BootDeviceType_0 | 0
BOOT_OPTION_CFG_DATA_2.ImageType_2 | 0x1E
BOOT_OPTION_CFG_DATA_2.BootDeviceType_2 | 0
```

Reply:
It would be clear to only change TCC related config (flag and image type) in CfgData_Tcc_Feature.dlt.
If need change boot media order or others, we had better change it in CfgData_BootOption.yaml.
For TCC support, it looks good to me. maybe you could create another patch to change the device boot order if required.

Reply:
Okay. Let's put TCC only things here.

Reply:
ok so user only need to change the boot priority in order to enable TCC